cmd/action/mux.go

```diff
@@ -102,9 +102,7 @@ func SchemaNameFromDSN(url string) (string, error) {
 func postgresSchema(dsn string) (string, error) {
 	url, err := url.Parse(dsn)
 	if err != nil {
-		// For backwards compatibility, we default to "public" when failing to
-		// parse.
-		return "public", nil
+		return "", nil
 	}
```

Comment:
Why did you remove this?

Reply:
no need anymore, now that we inspect the realm there is no need to define the "default schema" to inspect ( it works)